### PR TITLE
Adding leaky_relu torch frontend

### PR DESCRIPTION
**Pull request by AymenSe**

### CODE REVIEW

**Comment by AymenSe** [comment]
#2723

**Comment by AymenSe** [comment]
@bicycleman15 Could you please review the changes please

**Comment by bicycleman15** [comment]
Hey, @AymenSe it's a good start.
Could you please remove changes made at `.idea` folder, we don't want to touch that.
Also there are some merge conflicts, please resolve those as well.

Rest of the PR looks good to me. will merge as soon as you make the above changes 🙂

**Comment by AymenSe** [comment]
Hi, @bicycleman15  I think everything is okay now, could you check, please?

**Comment by bicycleman15** [comment]
Hey, I think the frontend tests are failing for `leaky_relu` that you implemented, could you please resolve it ? 🙂

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bicycleman15) [inline .idea/misc.xml:1]: You should add this file back, and it's contents should be exactly as in: https://github.com/unifyai/ivy/blob/master/.idea/misc.xml

File: .idea/runConfigurations/pytest_for_all__jax_.xml [hunk 2]
```diff
@@ -1,48 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```
Comment (bicycleman15) [inline .idea/runConfigurations/pytest_for_all__jax_.xml:1]: Same here for this file: add it back and the contents should be: https://github.com/unifyai/ivy/blob/master/.idea/runConfigurations/pytest_for_all__jax_.xml

**Review by bicycleman15 — CHANGES_REQUESTED** [review]
Have a look at my comments, the rest looks good. I will merge as soon as you do the changes :)

**Comment by AymenSe** [comment]
@bicycleman15 could you please check now?

**Review by bicycleman15 — APPROVED** [review]
LGTM!

**Pull request merged**